tools/run_integration_test.py

```diff
@@ -0,0 +1,315 @@
+#!/usr/bin/env python3
+"""A script that helps running integration tests for multiple Tizen plugins.
+
+To run integrations tests for all plugins under packages/, 
+run tools/run_integration_test.py in the root of this repository.
+
+Here are some of the common use cases. Assume packages contain plugins 
+a, b, c, d, and e.
+- running specific packages:
+tools/run_integration_test.py --plugins a b c # runs a, b, c
+- excluding some packages:
+tools/run_integration_test.py --exclude a b c # runs d, e
+- exclude has precedence:
+tools/run_integration_test.py --plugins a b c --exclude b # runs a c
+"""
+import argparse
+import subprocess
+import sys
+import os
+import time
+import re
+
+_TERM_RED = '\033[1;31m'
+_TERM_GREEN = '\033[1;32m'
+_TERM_YELLOW = '\033[1;33m'
+_TERM_EMPTY = '\033[0m'
+
+_LOG_PATTERN = r'\d\d:\d\d\s+([(\+\d+\s+)|(~\d+\s+)|(\-\d+\s+)]+):\s+(.*)'
+
+
+class PluginResult:
+    """A class that specifies the result of a plugin integration test.
+
+    Attributes:
+        run_state: The result of the test. Can be either succeeded for failed.
+        details: A list of details about the test result. (e.g. reasons for failure.)
+    """
+
+    def __init__(self, run_state, details=[]):
+        self.run_state = run_state
+        self.details = details
+
+    @classmethod
+    def success(cls):
+        return cls('succeeded')
+
+    @classmethod
+    def fail(cls, errors=[]):
+        return cls('failed', details=errors)
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(
+        description='A script to run multiple tizen plugin driver tests.')
+
+    parser.add_argument(
+        '--plugins',
+        type=str,
+        nargs='*',
+        default=[],
+        help='Specifies which plugins to test. If it is not specified and \
+            --run-on-changed-packages is also not specified, \
+                then it will include every plugin under packages.')
+    parser.add_argument('--exclude',
+                        type=str,
+                        nargs='*',
+                        default=[],
+                        help='Exclude plugins from test.')
+    parser.add_argument(
+        '--run-on-changed-plugins',
```

Comment:
```suggestion
        '--run-on-changed-packages',
```
Use consistent terms (line 62 and 80).